helm/k8s-initiator-app/templates/netpol.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ template "initiator.fullname" . }}
+  labels:
+    app: {{ template "initiator.fullname" . }}
+spec:
+  egress:
+    - {}
```

Comment:
Not sure what should actually be in here - this seems to work on the affected cluster.

Reply:
I guess nothing as this should not access the api server